server/src/internalClusterTest/java/org/opensearch/routing/AliasRoutingIT.java

```diff
@@ -509,7 +523,7 @@ public void testAliasSearchRoutingWithTwoIndices() throws Exception {
         }
         logger.info("--> verifying get with routing, should find");
         for (int i = 0; i < 5; i++) {
-            assertThat(client().prepareGet("alias-b1", "1").execute().actionGet().isExists(), equalTo(true));
+            assertBusy(() -> { assertThat(client().prepareGet("alias-b1", "1").execute().actionGet().isExists(), equalTo(true)); });
```

Comment:
nit: Overriding the assertThat inside OpenSearchIntegTestCase file would have helped all such cases, where in the first argument we have actual data fetch call.

Reply:
But, it also means that we wait for 10s on static values, which will increase the wait time.